pyopencl/__init__.py

```diff
@@ -1576,7 +1577,7 @@ def create_some_context(interactive: Optional[bool] = None,
             from pyopencl.tools import get_test_platforms_and_devices
             for _plat, devs in get_test_platforms_and_devices():
                 for dev in devs:
-                    return Context([dev])
```

Comment:
This code looks a bit strange, why not just return `Context([devs[0]])`?

Reply:
That wouldn't work if `devs` is empty?

Reply:
Oh, I see. Is that a Python pattern? I would just have written `if devs: return ...`

Reply:
I don't think it's a common pattern, but it avoids an `if` in Python. Not sure if that's much faster + this isn't a hot path..